Lib/email/utils.py

```diff
@@ -351,3 +324,11 @@ def localtime(dt=None, isdst=None):
     if dt is None:
         dt = datetime.datetime.now()
     return dt.astimezone()
+
+
+def __getattr__(attr):
+    # lazy import, to speedup module import time
```

Comment:
copyedit (I've not been so picky as to insert the definite article, though!):
```suggestion
    # lazy import, to speed up module import time
```